### ISSUE TITLE
Encryption of boolean false values returns nil

### ISSUE THREAD

**Issue by casiodk** [RailsEventStore/rails_event_store#754]
Hi guys.

I am using the encryption mapper, and noticed that when encrypting boolean false values it returns nil when decrypting. Have you come across this issue before?

**Comment by mpraglowski** [comment]
Nice find. I've prepared a failing test case for it and I could confirm the issue.

```
       Diff:

       @@ -1,5 +1,5 @@
       -#<RubyEventStore::Mappers::SomeEventWithPersonalInfo:0x00007f9e67156e88
       - @data={:personal_info=>false, :user_id=>123},
       +#<RubyEventStore::Mappers::SomeEventWithPersonalInfo:0x00007f9e578819e8
       + @data={:personal_info=>nil, :user_id=>123},
```

**Comment by mpraglowski** [comment]
Fixed and released in 1.1.1

**Comment by casiodk** [comment]
Thanks for the quick fix 👍
